README.md

```diff
@@ -63,6 +63,15 @@ former indicates in which application version the given database schema was intr
 
 [^1]: Semantic Versioning - https://semver.org/
 
+## How to measure code coverage
+```shell
+./mvn verify -Pcode-coverage
```

Comment:
We strongly prefer conditional profile activation (e.g. via `-D`) over the `-P`.

And in general, I guess this is a generic enough build feature that had better be included into the [AbsaOSS/root-pom](https://github.com/AbsaOSS/root-pom) repository.

Reply:
Refactored in commit 30320b0a59f4c2924653158fd2f8ef6a25d00062.